### PR TITLE
media-gfx/frogr: Version bump to 1.8.1

### PR DESCRIPTION
**Pull request by flint2**

### CODE REVIEW

**Comment by leio** [comment]
There's a 1.8.1 out a day after 1.8, which looks like a sort of a brown paperbag release we should go with instead at this point

**Comment by flint2** [comment]
> There's a 1.8.1 out a day after 1.8, which looks like a sort of a brown paperbag release we should go with instead at this point

Done in https://github.com/gentoo/gentoo/pull/34599/commits/40da1400dd1ae0c4070907125aff15b33dd44895

File: media-gfx/frogr/frogr-1.8.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+DESCRIPTION="flickr applications for GNOME"
+HOMEPAGE="https://live.gnome.org/Frogr"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=dev-libs/glib-2.56:2
+	>=dev-libs/json-glib-1.2
```
Comment (cfuga) [inline media-gfx/frogr/frogr-1.8.1.ebuild:17]: Upstream lowered the required version. It should be:
```
>=dev-libs/json-glib-1.1.2
```

File: media-gfx/frogr/frogr-1.8.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	>=dev-libs/libgcrypt-1.5:=
+	>=dev-libs/libxml2-2.6.8:2
+	media-libs/gstreamer:1.0
+	>=media-libs/libexif-0.6.14
+	>=net-libs/libsoup-2.34:2.4
```
Comment (cfuga) [inline media-gfx/frogr/frogr-1.8.1.ebuild:22]: Upstream bumped the default dependency for libsoup. It should be:
```
>=net-libs/libsoup-3.0:3.0
```

**Review by cfuga — CHANGES_REQUESTED** [review]
The new ebuild still needs two dependency changes.

**Comment by cfuga** [comment]
The last version looks fine for me. Thanks!

File: media-gfx/frogr/frogr-1.8.1.ebuild (hunk 2 above)
Comment (thesamesam) [reply media-gfx/frogr/frogr-1.8.1.ebuild]: Are you definitely always running `pkgdiff-mg` and reviewing release notes?

**Pull request closed without merging**